integration_testing/features/coach/coach-learners-report.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Coach learners report
+    Coach should be able to access Kolibri Coach tab
+    facility admin should be assign classes to coach facility to access the coach tab
+    Coach can look at each of the report types for the progress you have just logged
+
+  Background:
```

Comment:
Even though we are for now using these pickles just for manual testing, I wonder if we should still observe the conventions, both for our own educational purposes, and maybe in hope that one day we might still tackle writing the formal test steps and not have to rewrite the scenarios. So, how about we start recording the below values values in `Examples` anyways, and use the `<username>`, `<class>` and `<channel>` placeholders in the steps: 

```
Examples:
| username | class     | channel                    | topic         |
| Hansen   | Buffoons  | Pratham Books' StoryWeaver | Night Trouble |
```